### PR TITLE
ipnaint.html removed, inpaint.md added

### PR DESCRIPTION
**Pull request by ieliz**

### CODE REVIEW

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/inpaint.md [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+var lastY;
+
+window.onload = function()
+{
+context = document.getElementById("canvas").getContext("2d");
+img = document.getElementById("src");
+context.canvas.width = img.width;
+context.canvas.height = img.height;
+context.drawImage(img, 0, 0);
+};
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/inpaint.md:55]: Please fix indentation

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/inpaint.md [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+function init()
+{
+  let rgba = cv.imread("src");
+  src = new cv.Mat();
+  cv.cvtColor(rgba, src, cv.COLOR_RGBA2RGB);
+  mask = new cv.Mat.zeros(src.size(), cv.CV_8UC1);
+  rgba.delete();
+
+  cv.line(src, new cv.Point(0, 0), new cv.Point(Math.floor(src.cols * 0.5),
+  Math.floor(src.rows * 0.5)), new cv.Scalar(0, 0, 0, 255), 1);
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/inpaint.md]: Please rewrite using `for` loop and list of start and end positions

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/inpaint.md [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+
+function onOpenCvReady()
+{
+  document.getElementById("status").innerHTML = "OpenCV.js is ready.";
+  cv["onRuntimeInitialized"] = () =>
+  {
+    document.getElementById("myButton").disabled = false;
+    init();
+  }
+}
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/inpaint.md]: Please design it in the same way as we did in previous samples:
```html
 <script async src="opencv.js"
         onload="..."
         type="text/javascript">
```

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/inpaint.md [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+
+</script>
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/inpaint.md]: Can we partially hide all the methods except `inpaint`?

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/inpaint.md [hunk 5]
```diff
@@ -0,0 +1,250 @@
...
+<code>cv.line(cv.Mat for drawing, start point as new cv.Point(), end point as new cv.Point(), color as cv.Scalar(), thickness as a  number of pixels)</code>
+<code>cv.cvtColor(source, destination, method)</code>
+<code>cv.inpaint(source image, mask, destination, region of neighborhood as a number of pixels, method)</code>
+</section>
+
+## Instructions
+<section id='instructions'>
+In the current sample we using <code>cv.line()</code> function for damage picture with black lines. You should restore picture with inpaint function.
+Write a function which can draw a region the picture. Via using that function you can create mask for <code>cv.inpaint()</code> function. Then use "inpaint" function with source picture, created mask, and empty <code>cv.Mat()</code> for saving and showing result.
+</section>
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/inpaint.md:19]: User should call only `inpaint`

Comment (ieliz) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/inpaint.md:19]: Sentence was changed.

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/inpaint.md [hunk 6]
```diff
@@ -0,0 +1,213 @@
...
+  lastY = e.pageY - this.offsetTop;
+};
+
+document.getElementById("canvas").onmousemove = function(e)
+{
+  if (lastX != undefined)
+  {
+    let x = e.pageX - this.offsetLeft;
+    let y = e.pageY - this.offsetTop;
+    cv.line(src, new cv.Point(lastX, lastY), new cv.Point(x, y),new cv.Scalar(255, 255, 255, 255), 2);
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/inpaint.md]: Looks like we overoptimized the code so we draw on the same Mat which then is used in inpaint.

**Pull request merged**